src/edutap/wallet_google/api.py

```diff
@@ -130,8 +130,15 @@ def create(
         headers=headers,
     )
     if response.status_code == 403:
-        raise QuotaExceededException(
-            f"Quota exceeded while trying to create {name} {getattr(data, 'id', 'No ID')}"
+        # Check response text to distinguish between quota and permission errors
```

Comment:
i am not happy with duplicated code. This error handling need to bone im all api calls, shouldn't it be better to move it into an dedicated mothod and handled there properly? 

Also the response.text does hold additional information, why do not also add that at least as an log statement or in the Error Message